### ISSUE TITLE
How to change the default data type of a column?

### ISSUE THREAD

**Issue by Gao-Jun** [weshatheleopard/rubyXL#429]
I use RubyXL to generate a template xlsx file for users to input data.
For one column, say 'Student No.', some data start with 0, like `0011`.
I hope the user can input this directly without changing the data type of the column to text.

I know that I can change the data type of a cell, but the number of the data is various, so I don't know how many lines an user will input.

Is there any way to change the default type of a column?

Thank you.

**Comment by weshatheleopard** [comment]
Yes there is, you need to create a `col` element in `cols` collection on `worksheet` object with a respective `style` attribute.

Please do not expect me to explain everything. There's a lot of documentation available on XLSX format. Why people expect _me_ to solve _their_ problems?
